chore: update guidelines for making issues and commits (#553)
rupali-codes/LinksHub: CONTRIBUTING.md

#### 🌟Follow these steps to get your Links added into the Hub

* A newly created category and subcategory should be added to the [data.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/data.ts) & [index.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/index.ts) files

> * Each category has it's own folder with it's name

Reviewer: [Typo]
```suggestion
* Each category has its own folder with its name
```

> * Every subcategory should be placed in it's respective category folder

Reviewer: [Typo]
```suggestion
* Every subcategory should be placed in its respective category folder
```

> * Every subcategory should be placed in it's respective category folder

Reviewer: [Typo] see comment in [line 32](https://github.com/rupali-codes/LinksHub/pull/553/files#r1198230593)
```suggestion
* Every subcategory should be placed in it's respective category folder
```

> - Your entries should follow the above structure.

--- 
## Issues

> * Select an issue template from [issues](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) tab. Otherwise, choose **Other** if it doesn't match what you're looking for.

Author: ```suggestion
* Select an issue template from [issues](https://github.com/rupali-codes/LinksHub/issues/new/choose) tab. Otherwise, choose **Other** if it doesn't match what you're looking for.
```

> * Select an issue template from [issues](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) tab. Otherwise choose **Other** if it doesn't match what you're looking for.
> * When creating an issue make sure you fill up all the fields properly.
> * Make sure that you are NOT raising a **duplicate issue**.
> * If you want to work on the issue, please tick *I am willing to work on this issue* checkmark. Then you can start working on the issue.
> * If you aren't the owner of the issue, please comment that you're willing to work on the issue and wait for maintainers to assign you the issue. Also don't work on the issue if you're NOT assigned.
> * Work on only ONE issue at a time

Reviewer: [Typo]
```suggestion
* Select an issue template from [issues](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) tab. Otherwise, choose **Other** if it doesn't match what you're looking for.
* When creating an issue, make sure you fill up all the fields properly.
* Make sure that you are NOT raising a **duplicate issue**.
* If you want to work on the issue, please click on the *I am willing to work on this issue* checkmark. Then, you can start working on the issue.
* If you aren't the owner of the issue, please comment that you're willing to work on the issue and wait for maintainers to assign you the issue. Also, don't work on the issue if you're NOT assigned.
* Work on only ONE issue at a time. 
```

  
**Closing the issue**

> * While closing the issue, please leave a reasonable comment that Why you're closing the issue
> * If the Pull Request associated with issue get's mereged and the issue still remains open, it's you're responsibility to close the issue

Reviewer: [Typo}
```suggestion
* If you decide to close the issue, please leave a comment describing why(e.g., I'm busy with other obligations.) before you do. 
* **Note:** If the Pull Request associated with the issue gets merged and the issue still remains open, it's **your** responsibility to close the issue. 
```

## Commits

> * Please keep your commit message short and clear.

Reviewer: [Typo]
```suggestion
* Please keep your commit messages short and clear.
```

* Use `type: subject` format for writing your commit messages. `type` could be one the followings:
  * `feat`: use this if you're adding any new feature
  * `fix`: use this if you're fixing anything in the codebase
  * `chore`: use this when you're adding new links/resources AND when making any small changes

>   (ex. chore: added *resource_name* in *subcategory* *category* )
> 

Reviewer: [Proofreading] I think adding the link to conventional commits can be helpful if contributors are still stuck on how to write effective commits. 
```suggestion
  (ex. chore: added *resource_name* in *subcategory* *category* )
If you need more tips, check out [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
```

--- 
## Remarks ✅

> - If something is missing here, or you feel something is not well described, either create a PR, [raise an issue](https://github.com/rupali-codes/LinksHub/issues), or [do a code review of the person’s PR](https://www.freecodecamp.org/news/code-review-tips/)(ensure that your review conforms to the [Code of Conduct](https://github.com/CBID2/LinksHub-my-version-/blob/main/CODE_OF_CONDUCT.md)

Author: ```suggestion
- If something is missing here, or you feel something is not well described, either create a PR, [raise an issue](https://github.com/rupali-codes/LinksHub/issues), or [do a code review of the person’s PR](https://www.freecodecamp.org/news/code-review-tips/) (ensure that your review conforms to the [Code of Conduct](https://github.com/CBID2/LinksHub-my-version-/blob/main/CODE_OF_CONDUCT.md))
```
